priv/repo/migrations/20210729175737_add_slack_team_to_companies.exs

```diff
@@ -0,0 +1,216 @@
+defmodule ChatApi.Repo.Migrations.AddSlackTeamToCompanies do
+  use Ecto.Migration
+  import Ecto.Query, warn: false
+  require Logger
+  alias ChatApi.{Companies, Repo, SlackAuthorizations, SlackConversationThreads}
+  alias ChatApi.Companies.Company
+  alias ChatApi.SlackAuthorizations.SlackAuthorization
+  alias ChatApi.SlackConversationThreads.SlackConversationThread
+
+  def change do
+    alter table(:companies) do
+      add(:slack_team_id, :string)
+      add(:slack_team_name, :string)
+    end
+
+    alter table(:slack_conversation_threads) do
+      # TODO: try to backfill these fields based on existing data
```

Comment:
(may need to add a mix task to handle validating slack channel support per authorization)

Reply:
(can handle this in a future PR if necessary)